### PR TITLE
Support orthographic projection and a key toggle

### PR DESCRIPTION
**Pull request by skywhale**
For https://github.com/bschwind/opencascade-rs/issues/61

### CODE REVIEW

File: crates/viewer/src/camera.rs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                100.0 * self.aspect_ratio,
+                -100.0,
+                100.0,
+                -1000.0,
+                1000.0,
+            ),
+            Projection::Perspective => {
+                Mat4::perspective_rh(std::f32::consts::PI / 2.0, self.aspect_ratio, 0.01, 1000.0)
+            },
+        };
```
Comment (strohel) [inline crates/viewer/src/camera.rs:46]: I see that it is preexisting, but where the 1000 (and 100) constant comes from?

**Review by strohel — APPROVED** [review]
Nice! Just a curiosity question (independent from merging)

**Review by tuna-f1sh — APPROVED** [review]
Checked it out and works as I would expect. Switching between the two with the hidden lines visible it is like the infinite stairs illusion - I thought it was flipping at first!

File: crates/viewer/src/camera.rs (hunk 1 above)
Comment (skywhale) [reply crates/viewer/src/camera.rs:46]: This is a random constatnt so we can see the wholeness of the particular model we are loading. Once we have "scale to fit" function, we should be able to get rid of it and compute it more dynamically. Let me add a documentation.

**Comment by skywhale** [comment]
Thank you for the review @tuna-f1sh @strohel :) Added a clarying comment and merging.

**Comment by bschwind** [comment]
@skywhale I'd like to review it too if you can wait a bit before merging

**Comment by skywhale** [comment]
@bschwind no problem :) thank you!

**Review by bschwind — APPROVED** [review]
LGTM! I was going to suggest caching the camera matrix instead of just the aspect ratio but the performance overhead is so trivial it's not worth doing. And we used to calculate it every frame before anyway.

**Pull request merged**